src/lib/pipeline_configuration.py

```diff
@@ -534,6 +541,69 @@ def validate(self):
         validators.validate_string(self.table_name, "table_name")
 
 
+class TimestampRemapping(object):
```

Comment:
Is this being removed from anywhere?

Reply:
The object is not. It allows us to replace the body of TranslateRapidProKeys.remap_radio_shows.